pkg/oauthflow/flow.go

```diff
@@ -57,32 +56,36 @@ func ConnectorIDOpt(prov string) oauth2.AuthCodeOption {
 // DefaultIDTokenGetter is the default implementation.
 // The HTML page and message printed to the terminal can be customized.
 var DefaultIDTokenGetter = &InteractiveIDTokenGetter{
-	MessagePrinter: func(url string) { fmt.Fprintf(os.Stderr, "Your browser will now be opened to:\n%s\n", url) },
-	HTMLPage:       soauth.InteractiveSuccessHTML,
+	HTMLPage: soauth.InteractiveSuccessHTML,
+	Input:    os.Stdin,
```

Comment:
Can we make it so a `nil` input falls back to `os.Stdin`? Then each of the token getter vars below doesn't have to set it, and any new one doesn't have to remember to set it.

Reply:
Done!